src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const FIRST_LVL_DISCOUNT = 20;
+  const SECOND_LVL_DISCOUNT = 50;
+  const DAILY_PRICE = 40;
+  let totalValue = DAILY_PRICE * days;
+
+  if (days >= 3 && days < 7) {
+    totalValue = days * DAILY_PRICE - FIRST_LVL_DISCOUNT;
+  } else if (days >= 7) {
```

Comment:
pls avoid using magic numbers, move them to consts instead
<img width="806" alt="transOnVacChlist" src="https://github.com/mate-academy/js_task-transportation-on-vacation/assets/77551820/6a131fc3-d6a0-4ca0-92e4-084d003c1d40">